src/engraving/style/style.cpp

```diff
@@ -113,6 +113,16 @@ int MStyle::defaultStyleVersion() const
     return styleI(Sid::defaultsVersion);
 }
 
+void MStyle::setPreset(ScoreStylePreset preset)
+{
+    m_preset = preset;
```

Comment:
You can do this in the `.h` file since it's a trivial one-liner. That will improve performance since callers `#include` the header directly, so they can see the function definition at compile time and no longer need to look it up at runtime.

Doing that with a complicated function would slow compilation of files that include the header, but for a trivial function it makes no difference. The files would have to be recompiled each time the function is changed, but we're unlikely to change trivial functions like this very often.